db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_06_09_093354) do
+ActiveRecord::Schema.define(version: 2021_06_14_164900) do
```

Comment:
We have not added any migration in this PR. Why it is changing?

Reply:
I think this is due to the rebase, maybe I retained a different version number when I did a previous rebase from develop to auth-backend branch. After seen that issue in other rebases, I now try to keep the latest version number when rebasing.